dev/Repeater/ItemsRepeater.cpp

```diff
@@ -33,6 +33,11 @@ ItemsRepeater::ItemsRepeater()
         m_viewportManager = std::make_shared<ViewportManagerDownLevel>(this);
     }
 
+    if (!Layout())
+    {
+        Layout(winrt::StackLayout());
+    }
+
```

Comment:
We discussed moving this later -- perhaps we wrap the Layout() getter into a helper that fetches Layout() or the default StackLayout instance? We should try to defer it until at least layout begins so that we give Styles a chance to apply their values.

@ranjeshj can you advise?

Reply:
Layout() is used in the constructor just a few lines down, so I couldn't really move it without doing more surgery.

Reply:
Unfortunately without more surgery I suspect we will break scenarios like NavigationView where ItemsRepeater is having its layout set from a template. Or can you maybe show that we didn't break that scenario?

Reply:
Also reading Layout property from constructor seems weird -- the app has no reference to the object yet so they have no way even from code of setting the value.

Reply:
Well, I agree... I didn't have it there to begin with. :P

Reply:
@MikeHillberg Do you have a recommendation on how to set a "default" value in a constructor while allowing styles to override?

Reply:
Today we rely on the default style to initialize dependency properties, rather than the constructor. Setting it in the constructor makes it a local value, which will override any style values, thus breaking styling.